diff --git a/node_modules/@scalar/api-reference-react/dist/index-BLG_0Flx.cjs b/node_modules/@scalar/api-reference-react/dist/index-BLG_0Flx.cjs
index 3358d0a..905561f 100644
--- a/node_modules/@scalar/api-reference-react/dist/index-BLG_0Flx.cjs
+++ b/node_modules/@scalar/api-reference-react/dist/index-BLG_0Flx.cjs
@@ -1,4 +1,4 @@
-require('./index.css');"use strict";const l0=require("react");function wS(t){return t&&t.__esModule&&Object.prototype.hasOwnProperty.call(t,"default")?t.default:t}var vv={exports:{}},sf={};/**
+"use strict";const l0=require("react");function wS(t){return t&&t.__esModule&&Object.prototype.hasOwnProperty.call(t,"default")?t.default:t}var vv={exports:{}},sf={};/**
  * @license React
  * react-jsx-runtime.production.min.js
  *
diff --git a/node_modules/@scalar/api-reference-react/dist/index-BntBK5xT.js b/node_modules/@scalar/api-reference-react/dist/index-BntBK5xT.js
index 601ad98..44209ea 100644
--- a/node_modules/@scalar/api-reference-react/dist/index-BntBK5xT.js
+++ b/node_modules/@scalar/api-reference-react/dist/index-BntBK5xT.js
@@ -1,4 +1,3 @@
-import './index.css';
 import E$, { useRef as p9, useEffect as m9 } from "react";
 function ES(t) {
   return t && t.__esModule && Object.prototype.hasOwnProperty.call(t, "default") ? t.default : t;
